src/hotspot/os/bsd/os_bsd.cpp

```diff
@@ -2039,16 +2039,25 @@ jint os::init_2(void) {
     if (status != 0) {
       log_info(os)("os::init_2 getrlimit failed: %s", os::strerror(errno));
     } else {
-      nbr_files.rlim_cur = nbr_files.rlim_max;
-
-#ifdef __APPLE__
-      // Darwin returns RLIM_INFINITY for rlim_max, but fails with EINVAL if
-      // you attempt to use RLIM_INFINITY. As per setrlimit(2), OPEN_MAX must
-      // be used instead
-      nbr_files.rlim_cur = MIN(OPEN_MAX, nbr_files.rlim_cur);
-#endif
+      rlim_t rlim_original = nbr_files.rlim_cur;
+
+      // on macOS according to setrlimit(2), OPEN_MAX must be used instead
```

Comment:
Nit typo: s/on macOS/On macOS/

Reply:
That was deliberate, I tried to keep the same capitalization scheme as the other immediate code around this change and since they use small letters.

However, looking at the larger picture of the code around it, it appears that the rest capitalizes the comments.

I will switch it.